features/admin.actions.v1/package.json

```
@@ -15,6 +15,7 @@
         "@wso2is/admin.claims.v1": "^2.27.125",
         "@wso2is/admin.extensions.v1": "^2.38.2",
         "@wso2is/admin.feature-gate.v1": "^1.6.2",
+        "@wso2is/admin.organizations.v1": "^2.26.205",
```

let's use workspace protocol.

```suggestion
        "@wso2is/admin.organizations.v1": "workspace:^",
```

---

Fixed with https://github.com/wso2/identity-apps/pull/8687/commits/dc253d1fe5b9f554216212eb186caf0a30c4cba0